Topics/Development_Process/Vue_Test_Utils.md

```diff
@@ -0,0 +1,55 @@
+# Intro to Vue Test Utils (VTU)
+Vue Test Utils (commonly abbreviated as VTU) are a set of functions that make it easier to test Vue.js components.  It is commonly used
+with a test runner for automated testing in software engineering projects.
+
+## Why should I care?
+Automated testing is **essential** for any software engineering team.  With automated testing, teams can ensure they ship the most reliable
+and robust software to their clients ([Source](https://www.atlassian.com/continuous-delivery/software-testing/automated-testing)).  It can
+also provide confidence to teams whenever they make changes to the codebase in that previous software functionality will not suddenly break. 
+In a scenario where a developer could make a hard-to-detect breaking change to the codebase, automated testing can catch it before it becomes
+a much bigger (*and likely costly*) problem to the team.
+
+Although it is possible to test Vue components without the VTU, it becomes significantly more complex and error prone.
+The VTU makes it *much* easier to test your Vue.js components, especially when paired with a popular test runner like Jest.
```

Comment:
Similar to the previous comment, you should consider incorporating some concrete examples to demonstrate your point in this section.